apps/web/.env.example

```
@@ -2,3 +2,4 @@ API_INTERNAL_URL=http://localhost:4000
 API_PROXY_TIMEOUT_MS=30000        # upper bound per /api/backend/* request; a timeout returns a 504 envelope
 TRUST_PROXY=0                     # number of proxies you control between the internet and the web app; true means 1 and false means 0. Decides which X-Forwarded-For entry the proxy passes to the API. At 0 the API throttles every browser client as one connection. WARNS at startup in production when 0
 STATUS_PAGE_URL=                  # optional; an http or https status page hosted away from this machine. Set it and every error state links to it, leave it empty and nothing is shown. Anything that is not an http origin is ignored rather than linked
+SENTRY_DSN=                       # optional. Sentry dsn for scrubbed browser and server error reports. Read at runtime and only used when NODE_ENV=production. No Compose file sets it, so a self hosted deployment reports nothing until its operator opts in with a dsn of their own
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_DSN key should go before the STATUS_PAGE_URL key

(UnorderedKey)